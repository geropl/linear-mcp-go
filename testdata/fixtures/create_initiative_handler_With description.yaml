---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 348
        transfer_encoding: []
        trailer: {}
        host: api.linear.app
        remote_addr: ""
        request_uri: ""
        body: '{"query":"\n\t\tmutation InitiativeCreate($input: InitiativeCreateInput!) {\n\t\t\tinitiativeCreate(input: $input) {\n\t\t\t\tsuccess\n\t\t\t\tinitiative {\n\t\t\t\t\tid\n\t\t\t\t\tname\n\t\t\t\t\tdescription\n\t\t\t\t\turl\n\t\t\t\t}\n\t\t\t}\n\t\t}\n\t","variables":{"input":{"name":"Created Test Initiative 2","description":"Test Description"}}}'
        form: {}
        headers:
            Content-Type:
                - application/json
        url: https://api.linear.app/graphql
        method: POST
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: |
            {"data":{"initiativeCreate":{"success":true,"initiative":{"id":"c6a7dd0c-cbe2-4101-906d-ddd97acb2241","name":"Created Test Initiative 2","description":"Test Description","url":"https://linear.app/linear-mcp-go-test/initiative/created-test-initiative-2-e209008074dc"}}}}
        headers:
            Alt-Svc:
                - h3=":443"; ma=86400
            Cache-Control:
                - no-store
            Cf-Cache-Status:
                - DYNAMIC
            Content-Type:
                - application/json; charset=utf-8
            Etag:
                - W/"10e-Z2O6jMzejJ/UoHu3z6jnwHPqPzc"
            Server:
                - cloudflare
            Vary:
                - Accept-Encoding
            Via:
                - 1.1 google
        status: 200 OK
        code: 200
        duration: 0s
