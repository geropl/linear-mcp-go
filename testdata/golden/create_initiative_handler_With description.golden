err: ""
output: |
    Initiative: Created Test Initiative 2
      ID: c6a7dd0c-cbe2-4101-906d-ddd97acb2241
      Description: Test Description
      URL: https://linear.app/linear-mcp-go-test/initiative/created-test-initiative-2-e209008074dc
